translations/chi/get5.phrases.txt

```diff
@@ -226,7 +230,7 @@
     }
     "ChangingMapInfoMessage"
     {
-        "chi"            "切换地图至{GREEN}{1}..."
+        "chi"            "切换地图至{GREEN}{1}......"
```

Comment:
What's with the three extra periods?

Reply:
In Chinese, the ellipsis is six periods.

Reply:
Fair enough